MyCryptoHQ/MyCrypto
# Fix ens subdomain resolution

Closes #1977

### Description

Previous fix to #1977 was reverted due to an issue. This PR fixes an issue that doesn't allow subdomains to be resolved. It does NOT resolve the issue with `eth` being in the root domain, making it un-resolve-able.

### Changes

* Fixed a bug resolution did not take into account subdomain being present.

### Steps to Test

1. Navigate to the `/account` and select the *View Address* button.
2. Enter `compound.harrydenley.eth`. This should resolve to a non-zero address

### Screenshots
![snip](https://user-images.githubusercontent.com/29407814/51441293-125ce880-1c9e-11e9-83a1-3c58211015fe.PNG)
